src/play_time.c

```diff
@@ -45,6 +46,7 @@ void PlayTimeCounter_Update(void)
 
     gSaveBlock2Ptr->playTimeVBlanks = 0;
     gSaveBlock2Ptr->playTimeSeconds++;
+	FakeRtc_TickTimeForward();
```

Comment:
spaces instead of tabs

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/4910/commits/957dda9d1f078ae2fec18f1a7eaf881795225cb6